src/store.ts

```diff
@@ -73,7 +73,7 @@ export const createStore = ({
     edgeMeshes: [],
     selections,
     drags: {},
-    graph: new Graph(),
+    graph: new Graph({ multi: true }),
```

Comment:
Whats this do?

Reply:
Probably for cycle detection on certain layouts. Probably fine for now though just check all the layouts work as expected